src/main/scala/org/scalastyle/sbt/Plugin.scala

```diff
@@ -289,11 +292,14 @@ class SbtLogOutput[T <: FileSpec](config: Config, logger: Logger, warnError: Boo
     case StartFile(file) => logger.verbose("start file " + file)
     case EndFile(file) => logger.verbose("end file " + file)
     case StyleError(file, clazz, key, level, args, line, column, customMessage) => {
-      plevel(level)(location(file, line, column) + ": " +
-          Output.findMessage(messageHelper, key, args, customMessage))
+      if (!silent) {
+        plevel(level)(location(file, line, column) + ": " +
+            Output.findMessage(messageHelper, key, args, customMessage))
+      }
+    }
+    case StyleException(file, clazz, message, stacktrace, line, column) => {
+      if (!silent) logger.error(location(file, line, column) + ": " + message)
```

Comment:
In my case, yes, because I'm outputting them to a file. You have to turn on the silent option to make them  silent. It's not on by default.